src/gameData.js

```diff
@@ -151,7 +152,15 @@ const gameData = {
     maxUsers: 10,
     component: GoogleGossip
   },
-
+  SpaceInvaders: {
+    title: "Space Invaders",
+    authors: "Jason Alvarez, Patrick Cooney, David ",
+    description:
+      "Are you going to let Aliens invade your perfect dictatorship? Of course not! Defend your glorious planet.",
+    minUsers: 1,
+    maxUsers: 10,
+    component: SpaceInvaders
+  }
```

Comment:
Please only include this entry and the import line in this pull request (and not the other edits). Looks like the auto-formatter in Code Sandbox hates my style. :stuck_out_tongue: